src/styles/blocks/stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &-item {
+    width: 1rem;
+    height: $fs-basic;
```

should star's size depend on basic font size? I guess it's better to set it independently:
```suggestion
    width: 16px;
    height: 16px;
```
you also can use a variable